client/component/TimeSelect.jsx

```diff
@@ -6,10 +6,11 @@ const TimeSelect = ({ timeslots }) => (
     <img src='images/time.PNG' className={styles.icon}/>
     <select defaultValue='7:00 pm' className={styles.select}>
       {timeslots.map(timeSlot => {
-        if (timeSlot === '7:00 pm') {
-          return (<option selected key={timeSlot} value={timeSlot}>{timeSlot}</option>);
-        }
-        return (<option key={timeSlot} value={timeSlot}>{timeSlot}</option>);
+        let render;
+        timeSlot === '7:00 pm'
+          ? render = (<option selected key={timeSlot} value={timeSlot}>{timeSlot}</option>)
+          : render = (<option key={timeSlot} value={timeSlot}>{timeSlot}</option>);
+        return render;
```

Comment:
also an interesting way to render components that is different from our usual formula.

Reply:
cant return stuff in a ternary and I thought it looked cleaner than all the if statements
=